src/main/java/com/google/api/codegen/ApiConfig.java

```diff
@@ -112,11 +112,6 @@ private static String getPackageName(ConfigProto configProto) {
    * Returns the InterfaceConfig for the given API interface.
    */
   public InterfaceConfig getInterfaceConfig(Interface iface) {
-    InterfaceConfig interfaceConfig = getInterfaceConfigMap().get(iface.getFullName());
-    if (interfaceConfig == null) {
-      throw new IllegalArgumentException(
-          "no interface config for interface '" + iface.getFullName() + "'");
-    }
-    return interfaceConfig;
+    return getInterfaceConfigMap().get(iface.getFullName());
```

Comment:
Why remove the null check? Is getInterfaceConfigMap() now guaranteed to contain iface?